pkgs/top-level/all-packages.nix

```diff
@@ -11104,6 +11104,10 @@ with pkgs;
     postgresqlSupport = true;
   }).overrideAttrs { pname = "sqitch-pg"; };
 
+  sqitchSqlite = (callPackage ../development/tools/misc/sqitch {
+    sqliteSupport = true;
+  }).overrideAttrs { pname = "sqitch-sqlite"; };
```

Comment:
I don't see a point in having 3 of those. Why not just do sqitchFull and sqitch?

Reply:
Not entirely sure why it was done like this but it seems deliberate so I just followed it. I can only find this PR: https://github.com/NixOS/nixpkgs/pull/6895, and the discussion might have taken place somewhere else.

CC @robinbb @lucabrunox

Reply:
I think the only reason to have created 'sqitchPg' was that since, at the time, only PostgreSQL was supported, it did not make sense to call it 'sqitch'.

Withholding PR approval, waiting for @sekunho to implement the additional feature of creating 'sqitchFull' is not fair, however, and risks delaying/blocking/wasting the value of this PR.

Reply:
I see! Thank you @robinbb.

`sqitch` has a total of 9 targets (https://sqitch.org/download/), and `nixpkgs` does not contain all 9 of the necessary perl packages to implement it in its entirety. Perhaps we could stick with the current approach for now until it's possible to implement a full `sqitch` package?